lib/src/header/Header.tsx

```diff
@@ -120,30 +124,40 @@ const DxcHeader = ({
         {isResponsive && responsiveContent && (
           <MainContainer>
             <ChildContainer padding={padding}>
-              <HamburguerItem tabIndex={tabIndex} underlined={underlined} onClick={handleMenu}>
+              <HamburguerTrigger tabIndex={tabIndex} onClick={handleMenu}>
                 {hamburgerIcon}
-                <HamburguerTitle>Menu</HamburguerTitle>
-              </HamburguerItem>
+                Menu
```

Comment:
We need to also translate this text since it can't be customizable by the user.

Reply:
Done!